src/app.js

```diff
@@ -18,6 +18,12 @@ const providerConfig = {
       chainId: '0x03c401',
       rpcUrl: 'https://rpc.tau1.artis.network'
     },
+    {
+      name: 'volta',
+      chainId: '73799',
+      rpcUrl: 'https://volta-rpc.energyweb.org',
+      registry: '0xc15d5a57a8eb0e1dcbe5d88b8f9a82017e5cc4af',
```

Comment:
The contract is already deployed to the same address as on other networks, so the registry can also be omitted.
```suggestion
      registry: '0xdca7ef03e98e0dc2b855be647c39abe984fcf21b',
```

Reply:
Hi @mirceanis , I appreciate the review. That's great that you've been deploying the contracts to the various chains. Unfortunately (or fortunately as it means we've been using the `ethr` method for some time 😆 ), the contract at [0xc15d5a57a8eb0e1dcbe5d88b8f9a82017e5cc4af](https://volta-explorer.energyweb.org/address/0xC15D5A57A8Eb0e1dCBE5D88B8f9a82017e5Cc4AF/transactions) has already gotten a moderate amount of usage (looks like 23,537 transactions as of this message) so I think it would be good if folks with existing DIDs transactions on 0xc15d5a57a8eb0e1dcbe5d88b8f9a82017e5cc4af could resolve their DID document from the universal resolver. Do you think that would be okay or will this cause issues from your perspective?

I think that it will definitely make sense to use the standard `0xdca7ef03e98e0dc2b855be647c39abe984fcf21b` address for the Energy Web Chain (on which there isn't any `ethr` usage yet, to my knowledge)

Reply:
Oh, my mistake.. I hadn't checked the existing usage of the deployment you point to :)
That is fine, there should be no issues with using a different registry, however, if you want other folks to converge on the same registry for the Volta network, I think it should be noted in the official list of deployments: https://github.com/uport-project/ethr-did-registry#contract-deployments

I'm happy to hear it is already in use!

In this case, I think it's worth also adding a [test](https://github.com/uport-project/uport-did-driver/blob/master/src/__tests__/ethr.test.js).
With your config, these DIDs would look like this: 
`did:ethr:0x12047:0x02b97c30de767f084ce3080168ee293053ba33b235d7116a3263d29f1450936b71`
`did:ethr:volta:0x02b97c30de767f084ce3080168ee293053ba33b235d7116a3263d29f1450936b71`

About the other deployment, to be able to deploy the contract to the Energy Web chain, a small amount of tokens should be sent to the address "0x4580b7d2520c778DC4a1F305F53ee183B6E96746" (the same TX has cost 0.01950767 VT on the volta chain).
There is no known private key associated with that address so anything extra would remain locked there.
[This script](https://github.com/uport-project/ethr-did-registry/blob/develop/scripts/generateDeployTxs.js) is how the TX was generated without a key.

Reply:
@mirceanis I sent 0.3 EWT to "0x4580b7d2520c778DC4a1F305F53ee183B6E96746". Would you be able to execute the script to deploy the contract? Or should I be able to execute it myself?

Reply:
@jrhender, thank you. It is done: https://explorer.energyweb.org/tx/0xb0f3bc74f470f46f294ead3b625daf2216096db83c8169293a7294d7898b9d9d/raw-trace

You should also be able to execute it yourself using the script I linked to.
The script shows how the transaction is built and why there is no private key associated with that address.

Reply:
Thanks @mirceanis, you're the best 🙏 . I suppose we can add it to https://github.com/uport-project/ethr-did-registry#contract-deployments ? Glad to make the PR if you think it's a good idea 😄

Reply:
You're welcome!
And, your PR would be very welcome there :)
It's probably good to also set the official deployment for volta to the one you are already using